sql/009_join_business_with_perks.sql

```diff
@@ -0,0 +1,12 @@
+SELECT 
+sb.name business_name,
+ p.name AS perk_given, 
+ p.status,
+ p.starts,
+ p.ends
+FROM  perks p
+JOIN small_businesses sb
+    ON p.id = sb.id 
```

Comment:
Here is a problem

Let say we have perk with ID=1 which is entered by business A (the business A's ID=2).

We are never going to see this perk in this query. You need to use a foreign key instead. I sent you an example of a foreign key.

On top of that - we don't need `select` statements to be recorded alongside with DDL statements - but we can address it later.